docs/core/porting/third-party-deps.md

```diff
@@ -1,17 +1,18 @@
 ---
-title: Porting to .NET Core - Analyze your third-party dependencies
+title: Analyze dependencies to port to .NET Core
 description: Learn how to analyze third-party dependencies in order to port your project from .NET Framework to .NET Core.
 author: cartermp
 ms.author: mairaw
 ms.date: 12/04/2018
+ms.custom: seodec18
 ---
-# Analyze your third-party dependencies
+# Analyze your third-party dependencies to port code to .NET Core
```

Comment:
instead of third-party maybe we should call it external? Style guide asks us to avoid this word.

Reply:
Sounds good.